dev/TeachingTip/TeachingTip.cpp

```diff
@@ -793,7 +793,10 @@ void TeachingTip::OnIsOpenChanged()
             auto popup = winrt::Popup();
             m_popupOpenedRevoker = popup.Opened(winrt::auto_revoke, { this, &TeachingTip::OnPopupOpened });
             m_popupClosedRevoker = popup.Closed(winrt::auto_revoke, { this, &TeachingTip::OnPopupClosed });
-            popup.ShouldConstrainToRootBounds(ShouldConstrainToRootBounds());
+            if (SharedHelpers::Is19H1OrHigher())
```

Comment:
We shouldn't use platform checks, just use the QI of the interface like:
```c++
if (winrt::IPopup10 popup10 = popup)
{
    popup10.ShouldConstrainToRootBounds(ShouldConstrainToRootBounds());
}
```

The reason is that then if we're running on an older insider build that doesn't have this API yet, we don't crash. And it's way faster than an API information check.